greedy_algorithms/activity_selection.cpp

```diff
@@ -0,0 +1,42 @@
+// C++ program for activity selection problem.
+
+// The following implementation assumes that the activities
```

Comment:
please add proper documentation according to the contributing.md file

Reply:
Updated